### ISSUE TITLE
False whitespace error

### ISSUE THREAD

**Issue by thasner** [palantir/tslint#681]
Code:
```ts
let x = React.addons.renderIntoDocument(<SomeClass someProp={[]}/>);
```
Error:
```
/Users/thasner/git/compass/node_modules/gulp-tslint/node_modules/tslint/lib/tslint.js:43043
            this.limit = this.sourceFile.getFullWidth();
                                        ^
TypeError: Cannot read property 'getFullWidth' of undefined
    at EnableDisableRulesWalker.RuleWalker (/Users/thasner/git/compass/node_modules/gulp-tslint/node_modules/tslint/lib/tslint.js:43043:41)
    at EnableDisableRulesWalker.SkippableTokenAwareRuleWalker (/Users/thasner/git/compass/node_modules/gulp-tslint/node_modules/tslint/lib/tslint.js:43586:20)
    at new EnableDisableRulesWalker (/Users/thasner/git/compass/node_modules/gulp-tslint/node_modules/tslint/lib/tslint.js:43707:20)
    at Linter.lint (/Users/thasner/git/compass/node_modules/gulp-tslint/node_modules/tslint/lib/tslint.js:43976:31)
    at /Users/thasner/git/compass/node_modules/gulp-tslint/index.js:95:34
    at respond (/Users/thasner/git/compass/node_modules/gulp-tslint/node_modules/rcloader/index.js:68:7)
    at respond (/Users/thasner/git/compass/node_modules/gulp-tslint/node_modules/rcloader/node_modules/rcfinder/index.js:140:7)
    at next (/Users/thasner/git/compass/node_modules/gulp-tslint/node_modules/rcloader/node_modules/rcfinder/index.js:167:16)
    at process._tickCallback (node.js:355:11)
```

**Comment by JKillian** [comment]
I can't reproduce this. My guess is that gulp-tslint is using an old version of tslint which is unhappy with trying to run on a `.tsx` file.

**Comment by thasner** [comment]
I think you're right. This issue can likely be closed.
